cmd/config/internal/commands/sink_test.go

```diff
@@ -33,7 +33,7 @@ items:
     annotations:
       app: nginx2
       config.kubernetes.io/index: '0'
-      config.kubernetes.io/path: 'f1.yaml'
+      config.kubernetes.io/path: 'f1.json'
```

Comment:
rather than changing existing tests, would you create new ones specifically for json?

Reply:
@pwittrock Done.